tracardi/process_engine/action/v1/operations/contains_pattern/plugin.py

```diff
@@ -0,0 +1,152 @@
+from pydantic import validator
+import re
+
+from tracardi.process_engine.action.v1.operations.contains_pattern.patterns import patterns
+from tracardi.service.plugin.domain.register import Plugin, Spec, MetaData, Documentation, PortDoc, Form, FormGroup, \
+    FormField, FormComponent
+from tracardi.service.plugin.runner import ActionRunner
+from tracardi.service.plugin.domain.config import PluginConfig
+from tracardi.service.plugin.domain.result import Result
+
+
+class Config(PluginConfig):
+    field: str
+    pattern: str
+
+    @validator("field")
+    def validate_prefix(cls, value):
+        if value == "":
```

Comment:
"if not value" is gonna cover more cases

Reply:
one more thing: make this field an Enum, so it checks if it is one of email, ip, date, etc. even if the plugin is configured using advanced config.